### ISSUE TITLE
WC can reduce stock not respected by Ajax in backend

### ISSUE THREAD

**Issue by alexmigf** [woocommerce/woocommerce#26190]
**Is your feature request related to a problem? Please describe.**
I set this filter `woocommerce_can_reduce_order_stock` to false and orders coming from frontend respect it, but if i add a product from the backend this filter is not respected and the stock is changed.

**Describe the solution you'd like**
I would suggest add a filter condition in the Ajax method here: https://github.com/woocommerce/woocommerce/blob/dd49f89e77acaaa097251fe0a5dd69320ded48c6/includes/class-wc-ajax.php#L919

**Comment by woocommercebot** [comment]
Thanks for the suggestion @alexmigf.

We'll keep an eye on the [popularity of this request](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md#feature-requests) :)

**Comment by tammullen** [comment]
Hi @alexmigf 

Thank you for taking the time to share this idea, we really appreciate your help. 
​
While we appreciate you sharing your ideas with us, it doesn’t fit in with our current priorities for the project. At some point, we may revisit our priorities and look through the list of suggestions like this one to see if it warrants a second look. In the meantime, we are going to close this issue with the `votes needed` label and evaluate over time if this issue collects more feedback. 
​
Don’t be alarmed if you don’t see any activity on this issue for a while.
